Ghidra/Processors/6805/data/languages/6809.cspec

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<compiler_spec>
+  <global>
+    <range space="RAM"/>
+  </global>
+  <stackpointer register="S" space="RAM" growth="negative"/>
+  <default_proto>
+    <prototype name="__stdcall" extrapop="-2" stackshift="-2">
```

Comment:
If this isn't a particular compiler calling convention then should probably be:
 <prototype name="__stdcall" extrapop="2" stackshift="2" strategy="register">

Also, I think these values should be 2, which implies the 6605.cspec is wrong.

Reply:
Comparing with e.g. z80, 6502 or 68000 a value of 2 sounds reasonable. I will change it.
It looks like e.g. 6805.cspec or 8085.cspec using -2 are wrong. For both 6805 and 8085 the stack is growing towards lower memory addresses.

Reply:
Yes, it appears the 8085 and 6805 are wrong.  IMHO the stackshift and extrapop should always be a positive value, and the growth="positive" or "negative" would specify whether the value is added or subtracted from the stack pointer as part of a call.  Right now it is a bit confusing.